repository/model/repository_model.go

```diff
@@ -6,14 +6,35 @@ import (
 	"todo-clean/common"
 )
 
-type TbTodoRepositoryModel struct {
+type TbTodoRepositoryCreateModel struct {
 	ID          uuid.UUID `json:"id" gorm:"primary_key"`
 	Title       string    `json:"title"`
 	Description string    `json:"description"`
 	Status      string    `json:"status"`
 	CreatedAt   time.Time `json:"created_at"`
 }
 
-func (TbTodoRepositoryModel) TableName() string {
+type TbTodoRepositoryUpdateModel struct {
```

Comment:
Please define all update fields as pointer incase of we want to update some fields